app/controllers/categories_controller.rb

```diff
@@ -2,25 +2,38 @@ class CategoriesController < ApplicationController
   layout 'settings'
   before_action :set_category, only: [:edit, :update, :destroy]
   before_action :require_organization
+  before_action :redirect_for_not_ready_organization
 
   def index
     @categories = current_organization.categories.ordered.page(params[:page]).per(10)
   end
 
   def new
     @category = current_organization.categories.build
+    respond_to do |format|
+      format.html
+      format.js
+    end
   end
 
   def edit
   end
 
   def create
     @category = current_organization.categories.build(category_params)
-
     if @category.save
-      redirect_to categories_path, notice: 'Category was successfully created.'
+      respond_to do |format|
+        format.js
+        format.html do
+          redirect_to categories_path, notice: 'Category was
+            successfully created.'
+        end
+      end
     else
-      render action: 'new'
+      respond_to do |format|
```

Comment:
@dissident why do you use `respond_to` if you have same respond for both formats?